app/axiom/average.circuit.ts

```diff
@@ -29,10 +33,11 @@ export const circuit = async (inputs: CircuitInputs) => {
     throw new Error("Block number must be greater than the number of samples times the spacing");
   }
 
-  // Create an array of block numbers to sample from
```

Comment:
We should also validate `inputs.blockNumber.value() <= (samples * spacing)` in ZK